### PR TITLE
Add compat table process doc

### PR DESCRIPTION
**Pull request by ddbeck**
#### Summary

Document selection criteria and process for browsers in compat tables

#### Motivation

Mostly, to be transparently answer https://github.com/mdn/content/issues/12742, but also to make the whole process a bit less opaque and ad hoc.

This mostly recapitulates the process we last completed with the Deno project.

#### Related issues

* https://github.com/mdn/content/issues/12742
* https://github.com/mdn/browser-compat-data/pull/14629

#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by ddbeck** [comment]
@Rumyra click "details" for the Netlify deploy-preview in the checks list above, to see a fully built version of this.

File: meta/content/en/governance/browser-compatability.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  How we manage the contents of browser compatibility tables
+---
+
+Browser compatibility tables on MDN Web Docs are generated from [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) (also known as _BCD_). While MDN content and BCD are closely related projects, they're not directly tied to each other. This page describes the relationship between MDN content and BCD.
+
+## MDN Web Docs and BCD are distinct projects
+
+MDN Web Docs content is managed separately from browser compatibility data. BCD has its own [governance](https://github.com/mdn/browser-compat-data/blob/main/GOVERNANCE.md), [contribution process](https://github.com/mdn/browser-compat-data/blob/main/docs/contributing.md), and [content guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md).
+
+MDN uses browser compatibility to generate browser compatibility tables (plus some other site content). MDN uses the data as provided, including:
```
Comment (Rumyra) [inline meta/content/en/governance/browser-compatability.md:15]:
```suggestion
MDN uses browser compatibility data to generate browser compatibility tables (plus some other site content). MDN uses the data as provided, including:
```

File: meta/content/en/governance/browser-compatability.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. Complete the [browser selection questionnaire](#compatibility-table-browser-selection-questionnaire).
+
+   If you're comfortable sharing your responses publicly, do so;
+   otherwise, ask in the GitHub discussion for a Mozilla email contact.
+
+   If needed, the MDN team may ask follow-up questions.
+
+1. Wait for the team to consider your recommendation.
+
+Whether your your recommendation is selected, deferred, or rejected, we'll follow up in the GitHub discussion with next steps.
```
Comment (Rumyra) [inline meta/content/en/governance/browser-compatability.md:61]:
```suggestion
Whether your recommendation is selected, deferred, or rejected, we'll follow up in the GitHub discussion with next steps.
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

**Comment by Rumyra** [comment]
@ddbeck I reviewed - it all looks good to me. One thing was I couldn't seem to access a preview of the page via Details -> Netlify - there are preview links there, but not one for the bcd page 🤔 unless I'm missing something

**Comment by ddbeck** [comment]
> @ddbeck I reviewed - it all looks good to me. One thing was I couldn't seem to access a preview of the page via Details -> Netlify - there are preview links there, but not one for the bcd page 🤔 unless I'm missing something

Shows here for me: https://deploy-preview-13235--mdn-contributor-docs.netlify.app/governance/browser-compatability/ Does it not work for you?

**Comment by Rumyra** [comment]
> Shows here for me: https://deploy-preview-13235--mdn-contributor-docs.netlify.app/governance/browser-compatability/ Does it not work for you?

It does @ddbeck - I think I wasn't checking the right bcd link from the left navigation 🙈

**Review by Rumyra — APPROVED** [review]

**Pull request merged**